code/controllers/subsystem/minor_mapping.dm

```diff
@@ -42,15 +42,33 @@ SUBSYSTEM_DEF(minor_mapping)
 	var/turf_temperature = proposed_turf.temperature
 	return turf_gasmix.has_gas(/datum/gas/oxygen, 5) && turf_temperature < NPC_DEFAULT_MAX_TEMP && turf_temperature > NPC_DEFAULT_MIN_TEMP
 
-/datum/controller/subsystem/minor_mapping/proc/place_satchels(amount=10)
+/datum/controller/subsystem/minor_mapping/proc/place_satchels(amount=2)
```

Comment:
```suggestion
/datum/controller/subsystem/minor_mapping/proc/place_satchels(amount = 2)
```

Reply:
Actually scratch that, can you just set this to 0 and then called the place_satchels proc with 2?